### ISSUE TITLE
Improving mobile sidebar/home page. 

### ISSUE THREAD

**Issue by tobehonest** [fossasia/susi.ai#3461]
**Actual Behaviour**

![269D00F5-EA26-48FB-92A7-797201746006](https://user-images.githubusercontent.com/42002993/74301619-a43c0580-4d79-11ea-8b67-04024242ca26.jpeg)
![F3A97818-5D69-440C-8683-8D009A2CB3F0](https://user-images.githubusercontent.com/42002993/74301622-a736f600-4d79-11ea-9be8-92bc2af14fb2.jpeg)

**Expected Behaviour**

We must have either a sidebar or the categories on main page. 

**Suggestion**

1. We can have an always left button for mobile users
2. Or We can remove the side bar and keep the home page

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**
  Yes

@Dilshaad21  @Dishebh @plxity @anishagg17 @akshatnitd I need your suggestions on what are other possibilities on this.

**Comment by Dishebh** [comment]
@tobehonest yes, to me also it seems redundant to have both, the categories (at the bottom) and the side menu on the main page. Maybe we can remove the bottom menu, or the left menu (the left menu trigger button doesn't quite fit the website UI as well).

**Comment by adrijshikhar** [comment]
@tobehonest are you working on this?

**Comment by tobehonest** [comment]
> @tobehonest are stil you working on this?

This issue was fixed in the mentioned PR.

**Comment by adrijshikhar** [comment]
> > @tobehonest are stil you working on this?
> 
> This issue was fixed in the mentioned PR.

It is not fixed. The mentioned PR was closed not merged.

**Comment by tobehonest** [comment]
> > > @tobehonest are stil you working on this?
> > 
> > 
> > This issue was fixed in the mentioned PR.
> 
> It is not fixed. The mentioned PR was closed not merged.


Yes he made a new PR because of some issues with mentioned PR branch. You may navigate to susi.ai and see that sidebar button has been added.
